ops/Dockerfile

```diff
@@ -183,6 +183,18 @@ ENV CCACHE_DIR=/ccache
 
 RUN --mount=type=cache,target=/ccache/ source /conda/dev/bin/activate && ninja install
 
+ARG CCACHE_REMOTE_STORAGE
+RUN if [ -n "${CCACHE_REMOTE_STORAGE}" ] ; then \
+      apt-get update -qq && apt-get install -y ccache ; \
+      ccache --set-config=remote_only=true ; \
+      ccache --set-config=remote_storage=${CCACHE_REMOTE_STORAGE} ; \
+      ccache --set-config=log_file=/tmp/ccache.log ; \
```

Comment:
Why log to `/tmp` here?

Reply:
This is more like copy past here. 
If you are using different location and collecting statistic differently I can remove it.